Sources/Commands/SwiftPackageTool.swift

```diff
@@ -315,15 +316,17 @@ extension SwiftPackageTool {
                 throw InternalError("Could not get the current working directory")
             }
             
-            if localFileSystem.exists(AbsolutePath(url)) {
-                let currentTemplateLocation = AbsolutePath(url)
-                let templateDirectory = configPath.appending(components: "templates", "new-package", url)
-                
-                guard localFileSystem.exists(currentTemplateLocation) && localFileSystem.isDirectory(currentTemplateLocation) else {
-                    throw InternalError("Template folder \(url) could not be found in: \(cwd)")
+            if let path = try? AbsolutePath(validating: url) {
+                if localFileSystem.exists(path) {
+                    let currentTemplateLocation = path
+                    let templateDirectory = configPath.appending(components: "templates", "new-package", url)
+                    
+                    guard localFileSystem.exists(currentTemplateLocation) && localFileSystem.isDirectory(currentTemplateLocation) else {
+                        throw InternalError("Template folder \(url) could not be found in: \(cwd)")
```

Comment:
is this error message correct? Also this should not be an InternalError, as its a user error (bad input). use StringError instead.